packages/rocketchat-livechat/server/lib/QueueMethods.js

```diff
@@ -0,0 +1,114 @@
+RocketChat.QueueMethods = {
+	/* Least Amount Queuing method:
+	 *
+	 * default method where the agent with the least number
+	 * of open chats is paired with the incoming livechat 
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2726558665/issues/source?bid=3199634&fileBranchId=3348698#l5)